assets/blocks/window.block

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I am pretty sure there is a `glass` block which can used instead of adding a new block.

Reply:
Hey mentor(Naman-sopho) thanks for appreciating my design and thanks for viewing my work.
I have now removed the windows block and instead used Glass block from CoreBlocks